packages/graphile-utils/src/makeAddPgTableOrderByPlugin.ts

```diff
@@ -52,69 +52,57 @@ export default function makeAddPgTableOrderByPlugin(
   return plugin;
 }
 
+export interface OrderByAscDescOptions {
+  unique?: boolean;
+  nulls?: "first" | "last" | "first-iff-ascending" | "last-iff-ascending";
+}
+
 export function orderByAscDesc(
   baseName: string,
   columnOrSqlFragment: OrderBySpecIdentity,
-  unique = false,
-  options?: { createNullsFirst?: boolean; createNullsLast?: boolean }
+  uniqueOrOptions: boolean | OrderByAscDescOptions = false
 ): MakeAddPgTableOrderByPluginOrders {
-  const createNullsFirst = options?.createNullsFirst || false;
-  const createNullsLast = options?.createNullsLast || false;
+  const options =
+    typeof uniqueOrOptions === "boolean"
+      ? { unique: uniqueOrOptions }
+      : uniqueOrOptions ?? {};
+  const { unique = false, nulls = "last-iff-ascending" } = options;
```

Comment:
We should try to mirror (or, preferably, invoke) the logic here:

https://github.com/graphile/graphile-engine/blob/b715f6d1be7dd19e39830b4329fa3c758e5ae91b/packages/graphile-build-pg/src/plugins/PgConnectionArgOrderBy.js#L157-L164

Namely there's a `graphileBuildOptions.orderByNullsLast` setting that orders with nulls last always. So either factor that into the default:

```
const { unique = false, nulls = orderByNullsLast ? "last" : "last-iff-ascending" } = options
```

Or, preferably, allow `nulls` to be undefined and in that case go with the default (current) behaviour.